impl/src/main/java/org/ehcache/loaderwriter/writebehind/AbstractWriteBehindQueue.java

```diff
@@ -392,7 +386,7 @@ private void processItems() throws RuntimeException {
   private void processQuarantinedItems(List<SingleOperation<K, V>> quarantinedItems) throws Exception {
     LOGGER.debug("{} : processItems() : processing " + " quarantined items", getThreadName());
 
-    if (writeBatching && writeBatchSize > 0) {
+    if (writeBatchSize > 1) {
```

Comment:
I think this if block could be pushed deeper here, and then we can extract out the commonality between `processBatchedOperations(...)` and `processSingleOperation(...)`